draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.spec.tsx

```diff
@@ -155,4 +156,24 @@ describe("<VideoPlayer />", () => {
       expect(container.querySelector(".landscape")).toBeFalsy()
     })
   })
+
+  describe("Pausing/Playing animations", () => {
+    it("restarts the animation when pause button is clicked", () => {
+      const { container, getByRole } = render(
+        <VideoPlayer
+          fallback="illustrations/heart/spot/moods-cautionary.svg"
+          source="illustrations/heart/spot/moods-cautionary.webm"
+        />
+      )
+
+      const videoElement = container.querySelector("video")!
+
+      expect(videoElement.paused).toEqual(true)
```

Comment:
All HTMLVideoElement prototype functions have been mocked out by the previous test, so I can't really play a video in here - not sure it would even work in a Jest test.
There's a proper test for the hook though and this at least tests the overall wiring.